### PR TITLE
[README fix]

### PR DESCRIPTION
**Pull request by Yarboa**
README.developer.md link was broken
Some linting fixes added

### CODE REVIEW

**Review by mwperina — CHANGES_REQUESTED** [review]
Sorry, but converting all links from in-place to end-of-document doesn't make sense, it will just force you to click twice to get where you want

File: README.md [hunk 1]
```diff
@@ -1,34 +1,40 @@
...
 ## How to contribute
 
 ### Submitting patches
 
 Patches are welcome!
 
-Please submit patches to [github.com:containers/hirte](https://github.com/containers/hirte).
-More infomation about the development can be found in [README.developer](https://github.com/containers/hirte/README.developer.md).
+Please submit patches to [github.com:containers/hirte][3].
+More infomation about the development can be found in [README.developer](README.developer.md).
```
Comment (mwperina) [inline README.md]: Please just fix the link URL to:

```
[README.developer.md](README.developer.md)
```

Comment (Yarboa) [reply README.md]: Sure, 
Just run markdown linter and fixed some
as per this https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/about-writing-and-formatting-on-github

**Comment by rhatdan** [comment]
LGTM

**Comment by Yarboa** [comment]
Closing due to this  https://github.com/containers/hirte/pull/136 PR

**Pull request closed without merging**